cal/README.md

```diff
@@ -1,5 +1,22 @@
-# CAN Abstraction Layer (CAL)
+# CAL: CAN Abstraction Layer
```

Comment:
@Jchisholm204 Please update this README to reflect the new directory structure of CAL. Most of this can go into the decoder subdirectory.